media-gfx/tuxpaint/tuxpaint-0.9.24.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs xdg-utils desktop
+
+DESCRIPTION="Drawing program designed for young children"
+HOMEPAGE="http://www.tuxpaint.org/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="
+	app-text/libpaper
+	dev-libs/fribidi
+	gnome-base/librsvg:2
+	>=media-libs/libpng-1.2:0=
+	>=media-libs/freetype-2:2
+	media-libs/libsdl
+	media-libs/sdl-image[png]
+	media-libs/sdl-mixer
+	media-libs/sdl-pango
+	media-libs/sdl-ttf
+	x11-libs/cairo
+	"
+DEPEND="${RDEPEND}"
+BDEPEND="sys-devel/gettext
+	media-gfx/imagemagick[png]
+	dev-util/gperf
+	"
+
+src_prepare() {
+	default
+	sed -i \
+		-e 's|linux_ARCH_INSTALL:=install-xdg|linux_ARCH_INSTALL:=|' \
+		-e "s|linux_PREFIX:=/usr/local|linux_PREFIX:=/usr|" \
```

Comment:
I wonder if we need to split in a ${EPREFIX} before the /usr, actually.

Reply:
It'll take more than that to make this PREFIX-aware, I'm afraid. I fiddled with it minimally, and it doesn't quite work, there's always something that doesn't hit the right bucket.